core/database/foxx/tests/record.test.js

```diff
@@ -0,0 +1,341 @@
+"use strict";
+
+const chai = require("chai");
```

Comment:
Is there a reason we use Chai over Jest? 

I like Jest since it has builtin [sinon](https://github.com/sinonjs/sinon) functionality with [spyOn](https://jestjs.io/docs/jest-object#jestspyonobject-methodname) and [mockImplementation](https://jestjs.io/docs/mock-functions).

Reply:
Turns out it's cause ArangoDB comes pre-packaged with it, all good here